shimmingtoolbox/coils/biot_savart.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+# Code ported and refactored from Jason Stockmann and Fa-Hsuan Lin, "Magnetic field by Biot-Savart's Law"
+# http://maki.bme.ntu.edu.tw/?page_id=333
+
+import numpy as np
+
+MU0 = 1.256637e-6  # [H/m]
+H_GYROMAGNETIC_RATIO = 4.258e+7  # [Hz/T]
+
+
+def biot_savart(centers, normals, radii, segment_numbers, fov_min, fov_max, fov_n):
```

Comment:
for the unit test, we could have a unit test that checks a vector (hard-coded in the unit test), instead of needing to add a binary file for testing.

also, the test should be quick (ie: small number of segments, small matrix)